make/docker.mk

```diff
@@ -54,7 +54,7 @@ define DOCKER_BUILDX
 	# $(1) -- Image Tag
 	# $(2) -- Push (empty is no push | --push will push to dockerhub)
 	docker buildx build . --progress plain -t $(DOCKER_IMAGE_NAME):$(1) -f docker/Dockerfile.step-ca --platform="$(DOCKER_PLATFORMS)" $(2)
-	echo -n "$(COSIGN_PWD)" | cosign sign -key /tmp/cosign.key -r $(DOCKER_IMAGE_NAME):$(1)
+	cosign sign -r $(DOCKER_IMAGE_NAME):$(1)
```

Comment:
This looks like that will also sign the image with the `:latest`. Cosign help mentions that we should not do this:

```
Make sure to sign the image by its digest (@sha256:...) rather than by tag
(:latest) so that you actually sign what you think you're signing! This prevents
race conditions or (worse) malicious tampering.
```

Reply:
It looks like it's recommended for all tags, but it might be specially bad for `:latest`